src/main/java/frc/robot/pose/RobotPoseManager.java

```diff
@@ -1,25 +1,27 @@
 package frc.robot.pose;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import frc.robot.RobotConstants;
 
 public class RobotPoseManager {
 
-    private EncoderValues encoders;
+    private EncoderValues encoders = new EncoderValues(0, 0, 0, 0);
```

Comment:
this is a set of 4 magic numbers.
consider creating EncoderValues.NO_VALUES, so that the un-initialized values can be easily tested.